src/kernel.jl

```diff
@@ -509,6 +509,26 @@ moffat(α::Real, β::Real)                 = moffat(α, β, ceil(Int, (α*2*sqrt
     sum(x.^2)
 end
 
+"""
+butterworth(n,Wn,ls{tuple}) -> k
```

Comment:
indentation, and `::` for type annotation.

```suggestion
    butterworth(n, Wn, ls::Dims) -> k
```